custom/opm/reports.py

```diff
@@ -892,14 +891,19 @@ def row_objects(self):
         for index, row in enumerate(self.get_rows(self.datespan), 1):
             try:
                 case_row = self.get_row_data(row, index=1, awc_codes=awc_codes)
-                total_payment += case_row.cash_amt
-                rows.append(case_row)
-            except CaseOutOfRange:
-                try:
-                    rows.append(FakeConditionsMet(row, self, child_index=1, awc_codes=awc_codes))
-                except InvalidRow as e:
-                    if self.debug:
-                        self.add_debug_data(row._id, e)
+                if not case_row.case_is_out_of_range:
```

Comment:
@esoergel I exclude invalid rows in other reports.

I checked all code for reports and method ```get_row_data``` returns us the whole object and all properties calculated. If not we will get a traceback in method ```rows``` when we use specific property to get data. But now we only catch the InvalidRow  in ```row_objects```.